canonical-web-and-design/maas-ui
# fix(legacy): fix the events notification link to point to the right node

## Done

- Make the see all events notification link point to a device, machine or controller.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Remove this line so that the notifications always show: https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/app/partials/nodedetails/node-details-summary.html#L6
- Visit a machine details page, click on the "See all events" link in the notification. See that you get take to the events tab for that machine.
- Visit a device details page, click on the "See all events" link in the notification. See that you get take to the events tab for that device.
- Visit a controller details page, click on the "See all events" link in the notification. See that you get take to the events tab for that controller.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1773.
